app/routes/articles.js

```diff
@@ -24,12 +24,14 @@ router.post('/', isLoggedIn, hasAuthorRole, (req, res) => {
         return res.redirect('/')
     }
 
-    const {title, description, body} = req.body
-    const author = {id: req.user._id, username: req.user.username}
+    const title = encodeString(req.body.title)
```

Comment:
Maybe it would be safer to use middleware that sanitizes specific parameters in request, so we won't miss it in any endpoint?